homeassistant/components/google_assistant/trait.py

```diff
@@ -2765,7 +2765,13 @@ def query_attributes(self):
         if device_class is None or data is None:
             return {}
 
-        sensor_data = {"name": data[0], "rawValue": self.state.state}
+        try:
+            value = float(self.state.state)
+        except ValueError:
+            value = None
+        if self.state.state == STATE_UNKNOWN:
+            value = None
```

Comment:
We should have a tests for catching the valueerror and testing for unknown state .

Reply:
You are right, I'll add a test case

Reply:
It appeared there was also an error with `aqi` reporting, improved tests including an unknown state.